core/src/main/resources/org/apache/spark/ui/static/jquery.mustache.js

```diff
@@ -45,6 +45,19 @@
     return obj != null && typeof obj === 'object' && (propName in obj);
   }
 
+  /**
```

Comment:
These changes are all from the source jquery.mustache.js, not custom modifications

Reply:
I also checked that. This file is identical with `mustache.js` v3.0.1. Only the trailing spaces are removed.